modules/apps/frontend-data-set/frontend-data-set-views-web/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint'],
+	rules: {
+		'@typescript-eslint/naming-convention': [
+			'error',
+			{
+				format: ['PascalCase'],
+				prefix: ['I'],
+				selector: 'interface',
+			},
+		],
+	},
+};
```

Comment:
![](https://media1.giphy.com/media/1SjObB0Y07mg0/200.gif)

We can totally remove this too, I had been testing it out and then forgot about it. Figured it would be one less thing to think about in review for this module